packages/migrate/src/MigrateEngine.ts

```diff
@@ -43,6 +43,13 @@ export class MigrateEngine {
   private lastError: MigrateEngineLogLine['fields'] | null = null
   private initPromise?: Promise<void>
   private enabledPreviewFeatures?: string[]
+
+  // `latestSchema` is set to the latest schema that was used in `introspect()`
+  private latestSchema?: string
```

Comment:
Is this only used for Introspection? Then it might make sense to include that in the name maybe. `latestIntrospectedSchema` for example.

Reply:
Done